plugin/adsk/scripts/mayaUsd_pluginUICreation.mel

```diff
@@ -2,6 +2,7 @@ global proc mayaUsd_pluginUICreation()
 // Callback registered with the MFnPlugin for when the plugin is loaded.
 // Used to initialize any UI we create.
 {
+    source "USDMenuProc.mel";

```

Comment:
I'm a little concerned about moving this up to the top. That file uses the ufeSupport plugin which is only loaded below. I know this file only contains proc which are called on RMB, but by using ufeSupport it implies that it will already be loaded.

I would suggest putting your new global function hasPrimUpdater in this file (at the top). That way it will be available to any UI function that needs it such as UsdMenuProc and mayaUsdMenu - as both of these sourced below.